### DISCUSSION TITLE
I would like a mentor

### DISCUSSION THREAD

**Discussion by NEHA0901-US** [argoproj/argo-workflows#7863] [Q&A] [answered]
Hello ,
Myself Neha Sharma . I'm a pre-final Year B.E Computer Science  student from Pune University, India .
Right Now, I'm intern at IT company based in Banglore ,India as Front-End developer.
My previous Technical Internship was also as Front-end developer in a marketing agency Pune based company in India.
Tech Stack: React , Material UI , JavaScript , TypeScript(Familiar)

I'm Looking Forward to learn from Mentors and contribute with the best possible solution.

Have a Nice day.

**Comment by terrytangyuan** [comment, marked as answer]
Please submit an issue to request for mentor https://github.com/argoproj/argo-workflows/blob/master/docs/mentoring.md
